package.json

```diff
@@ -65,6 +65,7 @@
   },
   "devDependencies": {
     "@jupyterlab/builder": "^3.0.0",
+    "@jupyterlab/translation": "^3.0.0",
```

Comment:
This is a `dependencies` not a development dependency as the translation needs to be applied at runtime.